assembl/static2/js/app/components/common/richTextEditor/index.jsx

```diff
@@ -104,18 +106,20 @@ export default class RichTextEditor extends React.Component<Props, State> {
     };
 
     this.state = {
-      editorHasFocus: false
+      editorHasFocus: false,
+      editorState: initialEditorState
     };
   }
 
+  shouldComponentUpdate(nextProps: Props) {
+    const { initialEditorState } = this.props;
+    // Update the Editor only if the update is external
```

Comment:
I think it breaks the counter and the list of attachments. I have to do a lot of tests before to approve this

Reply:
il y a un test sur les conteurs et ça passe :)... mais oui, il faut vérifier si ça rentre pas en conflit avec d'autres besoins.

Reply:
Ok, counters work but it breaks reordering e.g. for resources administration :-(